yarn install throwing errors: error Command failed with exit code 1.

Hello devs, I saw you're making better UX ready admin. I just loved it & to explore it in deapth, I cloned and installed Yarn Dependencies as suggested in readme. Which have gone something unexpected. My bash history as follows: (Kindly help me, if I'm missing something) 

**For root folder:**
```
Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

PS D:\Playground\tabler> git clone https://github.com/tabler/tabler-react
Cloning into 'tabler-react'...
remote: Enumerating objects: 262, done.
remote: Counting objects: 100% (262/262), done.
remote: Compressing objects: 100% (262/262), done.
remote: Total 8668 (delta 199), reused 17 (delta 0), pack-reused 8406
Receiving objects: 100% (8668/8668), 42.06 MiB | 447.00 KiB/s, done.
Resolving deltas: 100% (5676/5676), done.
Checking out files: 100% (1401/1401), done.

PS D:\Playground\tabler> cd .\tabler-react\

PS D:\Playground\tabler\tabler-react> yarn install
yarn install v1.12.1
[1/4] Resolving packages...
[2/4] Fetching packages...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info fsevents@1.2.4: The platform "win32" is incompatible with this module.
info "fsevents@1.2.4" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning " > eslint-config-react-app@2.1.0" has incorrect peer dependency "babel-eslint@^7.2.3".
warning " > eslint-config-react-app@2.1.0" has incorrect peer dependency "eslint@^4.1.1".
warning " > eslint-config-react-app@2.1.0" has incorrect peer dependency "eslint-plugin-jsx-a11y@^5.1.1".
warning "react-styleguidist > react-docgen-displayname-handler@2.1.0" has incorrect peer dependency "recast@0.12.6 || 0.13.0".
warning "react-styleguidist > webpack-dev-server@2.11.2" has incorrect peer dependency "webpack@^2.2.0 || ^3.0.0".
warning "react-styleguidist > webpack-dev-server > webpack-dev-middleware@1.12.2" has incorrect peer dependency "webpack@^1.0.0 || ^2.0.0 || ^3.0.0".
[4/4] Building fresh packages...
$ yarn run build && yarn run copytablercsstodist
yarn run v1.12.1
$ yarn run copytablercsstodist && rollup -c && yarn run flow:copy-source
$ mkdir -p dist && cp src/Tabler.css src/Tabler.RTL.css dist/ && cp -r src/fonts src/images dist/
'cp' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

PS D:\Playground\tabler\tabler-react> yarn start
yarn run v1.12.1
$ yarn run copytablercsstodist && rollup -c -w
$ mkdir -p dist && cp src/Tabler.css src/Tabler.RTL.css dist/ && cp -r src/fonts src/images dist/
A subdirectory or file -p already exists.
Error occurred while processing: -p.
A subdirectory or file dist already exists.
Error occurred while processing: dist.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
PS D:\Playground\tabler\tabler-react>

```
and for **example folder** in separate terminal from VS Code.

```

Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

PS D:\Playground\tabler> cd .\tabler-react\example\
PS D:\Playground\tabler\tabler-react\example> yarn install
yarn install v1.12.1
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.2.4: The platform "win32" is incompatible with this module.
info "fsevents@1.2.4" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning " > react-google-maps@9.4.5" has unmet peer dependency "@types/googlemaps@^3.0.0".
warning " > react-google-maps@9.4.5" has unmet peer dependency "@types/markerclustererplus@^2.1.29".
warning " > react-google-maps@9.4.5" has unmet peer dependency "@types/react@^15.0.0 || ^16.0.0".
[4/4] Building fresh packages...

Done in 274.03s.
PS D:\Playground\tabler\tabler-react\example> yarn start
yarn run v1.12.1
$ react-scripts start
Starting the development server...
Failed to compile.

./src/HomePage.react.js
Module not found: Can't resolve 'tabler-react' in 'D:\Playground\tabler\tabler-react\example\src'
Compiling...

```

--- a/package.json
+++ b/package.json
@@ -24,9 +24,9 @@
     "styleguide": "styleguidist server",
     "styleguide:build": "styleguidist build",
     "commitmsg": "commitlint -e $GIT_PARAMS",
-    "copytablercsstodist": "mkdir -p dist && cp src/Tabler.css src/Tabler.RTL.css dist/ && cp -r src/fonts src/images dist/",
+    "copytablercsstodist": "mkdir -p dist && shx cp src/Tabler.css src/Tabler.RTL.css dist/ && shx cp -r src/fonts src/images dist/",
     "cm": "git-cz",
-    "startagain": "rm -rf dist node_modules example/node_modules yarn.lock example/yarn.lock && yarn install && cd example && yarn install && cd .. && yarn build"
+    "startagain": "shx rm -rf dist node_modules example/node_modules yarn.lock example/yarn.lock && yarn install && cd example && yarn install && cd .. && yarn build"
   },
   "lint-staged": {
     "src/**/*.js": "eslint --max-warnings=0"
@@ -80,6 +80,7 @@
     "rollup-plugin-postcss": "1.6.2",
     "rollup-plugin-url": "2.0.1",
     "semantic-release": "^15.5.0",
+    "shx": "^0.3.2",
     "webpack": "4.19.1"
   },
   "files": [

